hasher-matcher-actioner/webapp/src/Settings.jsx

```diff
@@ -3,11 +3,150 @@
  */
 
 import React from 'react';
+import Tabs from 'react-bootstrap/Tabs';
+import Tab from 'react-bootstrap/Tab';
 
 export default function Settings() {
   return (
     <>
-      <h1>Settings</h1>
+      <Tabs defaultActiveKey="signals" id="setting-tabs">
+        <Tab eventKey="signals" title="Signals">
+          <h2>Datasets</h2>
+          <div className="row mt-3">
+            <div className="col-xl-4 col-lg-4 col-md-6 col-sm-6 col-xs-12 mb-4">
```

Comment:
Usually, you just note the breakpoints. eg. 'col-lg-4, col-md-6', bootstrap figures the rest out on its own.